.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
```

Comment:
`Use this template for reporting a bug or a performance issue.`?

Reply:
done